### ISSUE TITLE
Recognize built-in php type

### ISSUE THREAD

**Issue by BestPig** [Gert-dev/php-integrator-linter#28]
Hello,

Just to warn about a little bug, maybe it is not really a bug, just a missing feature.

```
/**
 * @return integer
 */
```
-> integer was not found

The plugin check the presence of the class, but in this case it's a built-in php type, so it always exists.
You can have a list of existing types here : http://php.net/manual/fr/language.types.php

I don't known the right way to fix it, maybe it is not in the linter but it the integrator base ?

What do you think ?

**Comment by Gert-dev** [comment]
Thanks for contributing by reporting issues. Unfortunately, this one is not an actual bug but I agree it is confusing. `integer` is in fact a valid type if you look at the phpDocumentor documentation (it is an alias of `int`), but the draft PSR-5 standard does no longer list the alias. This was presumably done because the PHP 7 type is also `int` (similarly, the `boolean` type is also no longer valid, there is only `bool`).

In short, to fix this, you should use `int` instead of `integer` (and `bool` instead of `boolean`). This will also keep the docblocks consistent with PHP 7 scalar types if you decide to switch over.

**Comment by BestPig** [comment]
Thank you for this precise answer ;).

**Comment by Gert-dev** [comment]
Glad to be of help :+1:
